composer.json

```diff
@@ -32,9 +32,10 @@
         "friendsofphp/php-cs-fixer": "^2.16.1",
         "fzaninotto/faker": "~1.3",
         "pedrotroller/php-cs-custom-fixer": "^2.19",
-        "pedrotroller/symfony-integration-checker": "^2.0",
+        "pedrotroller/symfony-integration-checker": "^3.0",
         "phpspec/phpspec": "~6.0",
-        "sebastian/comparator": "^1.2.4"
+        "sebastian/comparator": "^1.2.4",
+        "symfony/var-dumper": "~4.3.7 || ~5.0"
```

Comment:
wha adding `symfony/var-dumper` ?

Reply:
Because of PHPStan.

Reply:
`symfony/var-dumper` is in the method signature of the DataCollectorInterface but not required in the composer.json of the http-kernel component.